### ISSUE TITLE
有没有文档啊 没有的话不容易看懂

### ISSUE THREAD

**Issue by tghjnjvg** [ahmedfgad/KivyAndroidStudio#1]
有没有文档啊 没有的话不容易看懂

**Comment by ahmedfgad** [comment]
Hi,
You can check my book [Building Android Apps in Python Using Kivy with Android Studio, Ahmed Fawzy Gad, Apress, October 2019, 978-1484250303](https://www.amazon.com/Building-Android-Python-Using-Studio/dp/1484250303). This GitHub project is discussed in its last chapter.

**Comment by tghjnjvg** [comment]
can i run kivy with android studio and write code on android studio, can you public kivy with gradle with android stuido

**Comment by ahmedfgad** [comment]
No, you cannot code Kivy in Android Studio. You can only create a Kivy application and then export an Android Studio project out of it.
